modular_citadel/code/modules/fish/fish_items.dm

```diff
@@ -0,0 +1,180 @@
+
+//////////////////////////////////////////////
+//			Aquarium Supplies				//
+//////////////////////////////////////////////
+
+/obj/item/weapon/egg_scoop
+	name = "fish egg scoop"
+	desc = "A small scoop to collect fish eggs with."
+	icon = 'modular_citadel/icons/obj/fish_items.dmi'
+	icon_state = "egg_scoop"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+
+/obj/item/weapon/fish_net
+	name = "fish net"
+	desc = "A tiny net to capture fish with. It's a death sentence!"
+	icon = 'modular_citadel/icons/obj/fish_items.dmi'
+	icon_state = "net"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+
+	suicide_act(mob/user)			//"A tiny net is a death sentence: it's a net and it's tiny!" https://www.youtube.com/watch?v=FCI9Y4VGCVw
+		to_chat(viewers(user), "<span class='warning'>[user] places the [src.name] on top of \his head, \his fingers tangled in the netting! It looks like \he's trying to commit suicide.</span>")
+		return(OXYLOSS)
+
+/obj/item/weapon/fishfood
+	name = "fish food can"
+	desc = "A small can of Carp's Choice brand fish flakes. The label shows a smiling Space Carp. Probably not made of carp meat."
+	icon = 'modular_citadel/icons/obj/fish_items.dmi'
+	icon_state = "fish_food"
+	throwforce = 1
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+
+/obj/item/weapon/tank_brush
+	name = "aquarium brush"
+	desc = "A brush for cleaning the inside of aquariums. Contains a built-in odor neutralizer."
+	icon = 'modular_citadel/icons/obj/fish_items.dmi'
+	icon_state = "brush"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+	attack_verb = list("scrubbed", "brushed", "scraped")
+
+	suicide_act(mob/user)
```

Comment:
Use absolute pathing instead of relative pathing for procs

Reply:
Necessary for the way suicide_act is handled, literally any time this is called it's done as mob/user

Reply:
Ditto